FV3/gfsphysics/GFS_layer/GFS_diagnostics.F90

```diff
@@ -3812,6 +3812,45 @@ subroutine GFS_externaldiag_populate (ExtDiag, Model, Statein, Stateout, Sfcprop
     ExtDiag(idx)%data(nb)%var2 => IntDiag(nb)%q_dt_int(:,5)
   enddo
 
+  idx = idx + 1
+  ExtDiag(idx)%axes = 2
+  ExtDiag(idx)%name = 'tsfc_via_diagnostics_manager'
```

Comment:
What about `_interval_average` instead of `_via_diagnostic_manager`? This would be describing what it is, rather than how it was computed.

Reply:
It's a little tricky since the diagnostics manager can produce interval average, interval minimum, interval maximum, or instantaneous diagnostics depending on how one configures the model.  Most physics diagnostics are not controlled by the diagnostics manager, and so are hard-coded to be average or instantaneous.  In that sense these are outliers, so I like to be explicit that it's the case.